haskell/repl.bzl

```diff
@@ -319,12 +319,12 @@ def _create_repl(hs, posix, ctx, repl_info, output):
             "%{ENV}": render_env(hs.env),
             "%{TOOL}": hs.tools.ghci.path,
             "%{CC}": hs.toolchain.cc_wrapper.executable.path,
-            "%{ARGS}": " ".join(
+            "%{ARGS}": "(" + " ".join(
                 args + [
                     shell.quote(a)
                     for a in quote_args
                 ],
-            ),
+            ) + ")",
```

Comment:
could be https://github.com/bazelbuild/bazel-skylib/blob/0.6.0/lib/shell.bzl#L17-L35, but looks like the same implementation.

Reply:
Yeah it could be, but as members `quote_args` are already quoted, we would need to change the preceding code to to be able to call `array_literal` on `args ++ quote_args`. This solution seemed simpler to me.